Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM nginx:1.25.2
+FROM nginx:1.25.2 AS opensafely-proxy
 ENV PORT=8080
 # Default orgs, must be pipe separated as are fed into a regex match
 # Can be overridden via run time env var
 ENV ORGS=opensafely|opensafely-core|opensafely-actions
 COPY *.conf.template /etc/nginx/templates/
-# uncomment to build a debug version locally
-#CMD ["nginx-debug", "-g", "daemon off; error_log /var/log/nginx/error.log debug;"]
+
+
+FROM proxy AS proxy-debug
```

Comment:
```suggestion
FROM opensafely-proxy AS opensafely-proxy-debug
```

Reply:
Have fixed the bug, but kept the local-only name as `proxy-debug`

Reply:
🤝